### ISSUE TITLE
Bluetooth data is not being recieved.

### ISSUE THREAD

**Issue by Spagett1** [joric/pinecil#16]
When i try to connect my pinecil v2 to this project i can connect yet no data is sent.

I believe this issue is due to my running this IronOs firmware https://github.com/Ralim/IronOS/actions/runs/4549363895  which uses 128 UUID

Screenshot:
![image](https://user-images.githubusercontent.com/77225642/228833315-89a4cf27-3cf4-4fd8-aec2-f3efb6da304c.png)

Browser: Google Chrome
OS: Artix Linux.

**Comment by River-Mochi** [comment]
also not working on main  Joric website 
https://joric.github.io/pinecil/

- PineSAM has updated code to fixes issues with  new 128 UUID  ble lables and is working now with newest  IronOS  that has 128 uuid. 

-  if  it helps could look at it to see changes needed in the code to work with new  IronOS.

**Comment by River-Mochi** [comment]
@joric

if it helps these were the two lines changed in  Tom's ESP-home for the 128 bit UUID 
<br>
![image](https://user-images.githubusercontent.com/97197236/230584134-e43cedc3-0715-48a2-912f-24d3d6fd529f.png)

**Comment by joric** [comment]
You can't really link to actions, they expire in a few days. So, what release version are you using?

**Comment by River-Mochi** [comment]
this version works  fresh today.   https://github.com/Ralim/IronOS/actions/runs/4637328253

Ralim is just waiting for last minute translations to come in and then will release this as 2.21  soon.

**Comment by River-Mochi** [comment]
> You can't really link to actions, they expire in a few days. So, what release version are you using?

I'm using 2.21.1E849AA   07-04-23

**Comment by joric** [comment]
I am sorry, it doesn't really seem to work. Anyone care for a working PR that actually works? Also my bluetooth explorer now crashes for some reason.

![image](https://user-images.githubusercontent.com/852547/230637843-f9a5fe9b-bc3e-438c-9416-85e2e5622791.png)

This doesn't work:

```diff
diff --git a/index.html b/index.html
index 15cbe3d..1f12b58 100644
--- a/index.html
+++ b/index.html
@@ -198,12 +198,12 @@ function init() {
     return false;
   }
           
-  BT_UUID_SVC_BULK_DATA                 = '9eae1adb-9d0d-48c5-a6e7-ae93f0ea37b0'; // bulk service
-  BT_UUID_CHAR_BLE_BULK_LIVE_DATA       = BluetoothUUID.canonicalUUID(1);
+  BT_UUID_SVC_BULK_DATA                        = '9eae1000-9d0d-48c5-aa55-33e27f9bc533'; // bulk service
+  BT_UUID_CHAR_BLE_BULK_LIVE_DATA              = 'd85ef001-168e-4a71-aa55-33e27f9bc533'; //BluetoothUUID.canonicalUUID(1);
 
-  BT_UUID_SVC_SETTINGS_DATA             = 'f6d75f91-5a10-4eba-a233-47d3f26a907f'; // settings service
-  BT_UUID_CHAR_BLE_SETTINGS_VALUE_SAVE  = BluetoothUUID.canonicalUUID("0xFFFF"); // write 1 to save settings
-  BT_UUID_CHAR_BLE_SETTINGS_VALUE_SETPOINT     = BluetoothUUID.canonicalUUID(0); // setpoint temp setting
+  BT_UUID_SVC_SETTINGS_DATA                    = 'f6d80000-5a10-4eba-aa55-33e27f9bc533'; // settings service
+  BT_UUID_CHAR_BLE_SETTINGS_VALUE_SAVE         = 'f6d7ffff-5a10-4eba-aa55-33e27f9bc533'; //BluetoothUUID.canonicalUUID("0xFFFF"); // write 1 to save settings
+  BT_UUID_CHAR_BLE_SETTINGS_VALUE_SETPOINT     = 'f6d70000-5a10-4eba-aa55-33e27f9bc533'; //BluetoothUUID.canonicalUUID(0); // setpoint temp setting
 
   autoconnect = typeof navigator.bluetooth.getDevices != 'undefined'
 
```

```sh
> Got 1 Bluetooth devices.
index.html:297 > Bluetooth Device connected.
index.html:320 DOMException: No Services matching UUID f6d80000-5a10-4eba-aa55-33e27f9bc533 found in Device.
index.html:335 DOMException: No Services matching UUID 9eae1000-9d0d-48c5-aa55-33e27f9bc533 found in Device.
```

I hate to link ephemeral links but it is this build https://github.com/Ralim/IronOS/actions/runs/4637328253

**Comment by River-Mochi** [comment]
maybe come into the live /pinecil-dev chat.  after discussions this morning  Builder555 PineSAM is working.  none of problems in pineSAM were from  IronOS. 

- Builder555  was able to fix  the bug this morning, basically a typo bc he had hard coded the UUID  in extra place he forgot about.  
- Tom had problem using wrong Save to flash UUID  which he corrected now but he still has another different problem too just connecting his pinecil to Bluetooth.

PineSam is 100% working. - except for  Screen brightness but that has nothing to do with  BLE.

**Comment by joric** [comment]
fixed in d90984d6. I had to delete it from windows devices and it worked. LE explorer also works.

**Comment by River-Mochi** [comment]
> I had to delete it from windows devices and it worked

🤣   windows !!  how does it keep remembering Pinecil and attaching it.
